### ISSUE TITLE
Adminhtml textarea field doesn't accept maxlength

### ISSUE THREAD

**Issue by rafaelstz** [magento/magento2#21779]
### Preconditions (*)
When I use *textarea* field type I can't use the attribute `maxlenght`, but when I use `text` I can use it.

File: **app/code/MyCompany/MyModule/Block/Adminhtml/Manage/Edit/Form.php**
### Actual result (*)
```php
$fieldset->addField(
    'notes',
    'textarea',
    [
        'label' => __('Notes'),
        'name' => 'notes',
        'maxlength' => '70'
    ]
);
### Expected result (*)
```

<img width="1279" alt="Screenshot 2019-03-15 15 53 21" src="https://user-images.githubusercontent.com/610598/54458480-7d74e980-473a-11e9-9136-75739e7c8ed1.png">

## Text

```php
$fieldset->addField(
    'notes',
    'text',
    [
        'label' => __('Notes'),
        'name' => 'notes',
        'maxlength' => '70'
    ]
);
```

<img width="1245" alt="Screenshot 2019-03-15 15 52 31" src="https://user-images.githubusercontent.com/610598/54458491-836aca80-473a-11e9-9442-092a654af12b.png">

**Comment by magento-engcom-team** [comment]
Hi @rafaelstz. Thank you for your report. 
To help us process this issue please make sure that you provided the following information:
- [x] Summary of the issue
- [ ] Information on your environment
- [x] Steps to reproduce
- [x] Expected and actual results

Please make sure that the issue is reproducible on the vanilla Magento instance following **Steps to reproduce**. To deploy vanilla Magento instance on our environment, please, add a comment to the issue: 

`@magento-engcom-team give me 2.3-develop instance` - upcoming 2.3.x release

For more details, please, review the [Magento Contributor Assistant](https://devdocs.magento.com/guides/v2.2/contributor-guide/contributing.html#contributor-assist) documentation.

@rafaelstz do you confirm that you was able to reproduce the issue on vanilla Magento instance following steps to reproduce?
- [x] yes
- [ ] no

**Comment by magento-engcom-team** [comment]
Hi @engcom-backlog-nazar. Thank you for working on this issue. 
In order to make sure that issue has enough information and ready for development, please read and check the following instruction: :point_down:
- [x] 1. Verify that issue has all the required information. (**Preconditions**, **Steps to reproduce**, **Expected result**, **Actual result**).<details><summary>Details</summary>If the issue has a valid description, the label `Issue: Format is valid` will be added to the issue automatically. Please, edit issue description if needed, until label `Issue: Format is valid` appears.</details>

- [x] 2. Verify that issue has a meaningful description and provides enough information to reproduce the issue. If the report is valid, add `Issue: Clear Description` label to the issue by yourself.

- [x] 3. Add `Component: XXXXX` label(s) to the ticket, indicating the components it may be related to.
- [x] 4. Verify that the issue is reproducible on `2.3-develop` branch<details><summary>Details</summary>- Add the comment `@magento-engcom-team give me 2.3-develop instance` to deploy test instance on Magento infrastructure. <br/> - If the issue is reproducible on `2.3-develop` branch, please, add the label `Reproduced on 2.3.x`.<br/>- If the issue is not reproducible, add your comment that issue is not reproducible and close the issue and **_stop verification process here_**!
</details>

- [x] 5. Verify that the issue is reproducible on `2.2-develop` branch. <details><summary>Details</summary>- Add the comment `@magento-engcom-team give me 2.2-develop instance` to deploy test instance on Magento infrastructure. <br/>- If the issue is reproducible on `2.2-develop` branch, please add the label `Reproduced on 2.2.x`</details>

-  _Next steps are available in case you are a member of **Community Maintainers**._

- [x] 6. Add label `Issue: Confirmed` once verification is complete. 

- [x] 7. Make sure that automatic system confirms that report has been added to the backlog.

**Comment by magento-engcom-team** [comment]
:white_check_mark: Confirmed by @engcom-backlog-nazar 
 Thank you for verifying the issue. Based on the provided information internal tickets `MAGETWO-98756`, `MAGETWO-98757` were created

**Issue Available**: @engcom-backlog-nazar, _You will be automatically unassigned. Contributors/Maintainers can claim this issue to continue. To reclaim and continue work, reassign the ticket to yourself._

**Comment by magento-engcom-team** [comment]
Hi @kisroman. Thank you for working on this issue. 
Looks like this issue is already verified and confirmed. But if you want to validate it one more time, please, go though the following instruction:
- [ ] 1. Add/Edit `Component: XXXXX` label(s) to the ticket, indicating the components it may be related to.

- [x] 2. Verify that the issue is reproducible on `2.3-develop` branch<details><summary>Details</summary>- Add the comment `@magento-engcom-team give me 2.3-develop instance` to deploy test instance on Magento infrastructure. <br/> - If the issue is reproducible on `2.3-develop` branch, please, add the label `Reproduced on 2.3.x`.<br/>- If the issue is not reproducible, add your comment that issue is not reproducible and close the issue and **_stop verification process here_**!
</details>

- [ ] 3. Verify that the issue is reproducible on `2.2-develop` branch. <details><summary>Details</summary>- Add the comment `@magento-engcom-team give me 2.2-develop instance` to deploy test instance on Magento infrastructure. <br/>- If the issue is reproducible on `2.2-develop` branch, please add the label `Reproduced on 2.2.x`</details>

- [ ] 4. If the issue is not relevant or is not reproducible any more, feel free to close it.

**Comment by magento-engcom-team** [comment]
Hi @rafaelstz. Thank you for your report. 
The issue has been fixed in magento/magento2#21816 by @kisroman in 2.3-develop branch
Related commit(s):
 - [1dc63e8becdceb7ec50560c30af14e74cdd355d1](https://github.com/magento/magento2/commit/1dc63e8becdceb7ec50560c30af14e74cdd355d1)
 - [4a0fb16a2fdd4082299cb14a9b0e1f5f48504b2c](https://github.com/magento/magento2/commit/4a0fb16a2fdd4082299cb14a9b0e1f5f48504b2c)
 - [149294b87a63ef89da774862d5ebcacee47975ff](https://github.com/magento/magento2/commit/149294b87a63ef89da774862d5ebcacee47975ff)

The fix will be available with the upcoming 2.3.2 release.
